samples/OpenIdConnectSample/Startup.cs

```diff
@@ -28,8 +29,10 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
             app.UseOpenIdConnectAuthentication(options =>
             {
                 options.ClientId = "63a87a83-64b9-4ac1-b2c5-092126f8474f";
+                options.ClientSecret = "Yse2iP7tO1Azq0iDajNisMaTSnIDv+FXmAsFuXr+Cy8=";
```

Comment:
nit: what's the point of setting `client_secret` if you're also using `response_type=id_token`?